rules/cloud/azure_network_security_modified_or_deleted.yml

```diff
@@ -0,0 +1,24 @@
+title: Azure Network Security Configuration Modified or Deleted
+id: d22b4df4-5a67-4859-a578-8c9a0b5af9df
+description: Identifies when a network security configuration is modified or deleted.
+author: Austin Songer
+status: experimental
+date: 2021/08/08
+references:
+    - https://docs.microsoft.com/en-us/azure/role-based-access-control/resource-provider-operations
+logsource:
+  service: azure.activitylogs
+detection:
+    selection:
+        properties.message:
+            - MICROSOFT.NETWORK/NETWORKSECURITYGROUPS/WRITE
+            - MICROSOFT.NETWORK/NETWORKSECURITYGROUPS/DELETE
+            - MICROSOFT.NETWORK/NETWORKSECURITYGROUPS/SECURITYRULES/WRITE
+            - MICROSOFT.NETWORK/NETWORKSECURITYGROUPS/SECURITYRULES/DELETE
```

Comment:
Not this ?
 - Microsoft.Network/networkSecurityGroups/join/action 
 - Microsoft.Network/networksecuritygroups/providers/Microsoft.Insights/diagnosticSettings/write